integration-cli/docker_api_inspect_test.go

```diff
@@ -20,26 +20,15 @@ func (s *DockerAPISuite) TestInspectAPIContainerResponse(c *testing.T) {
 	keysBase := []string{
 		"Id", "State", "Created", "Path", "Args", "Config", "Image", "NetworkSettings",
 		"ResolvConfPath", "HostnamePath", "HostsPath", "LogPath", "Name", "Driver", "MountLabel", "ProcessLabel", "GraphDriver",
+		"Mounts",
 	}
 
-	type acase struct {
+	cases := []struct {
 		version string
 		keys    []string
+	}{
+		{version: "v1.24", keys: keysBase},
 	}
-
-	var cases []acase
-
-	if testEnv.DaemonInfo.OSType == "windows" {
-		// Windows only supports 1.25 or later
-		cases = []acase{
-			{"v1.25", append(keysBase, "Mounts")},
-		}
-	} else {
-		cases = []acase{
-			{"v1.24", append(keysBase, "Mounts")},
-		}
-	}
-
 	for _, cs := range cases {
```

Comment:
Should we give this one a little cleanup? (maybe in future PRs?)

Reply:
Ah, yes, probably a later commit removed more. Sorry for that (that's sometimes the _downside_ of reviewing per-commit).

I did create a ticket to have more fixtures added (not sure if this is the same test, but I think it was related at least) https://github.com/moby/moby/pull/47155#discussion_r1462354403